src/main/java/com/internet/shop/controllers/product/DeleteProductController.java

```diff
@@ -21,6 +21,6 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
         String userId = req.getParameter("id");
         Long id = Long.valueOf(userId);
         productService.delete(id);
-        resp.sendRedirect(req.getContextPath() + "/product/all");
+        resp.sendRedirect(req.getContextPath() + "/product/all-for-admin");
```

Comment:
```suggestion
        resp.sendRedirect(req.getContextPath() + "/products/manage");
```
after changing the mapping be sure that you updated it everywhere you used it